spec.md

```diff
@@ -1601,6 +1655,72 @@ The CO MUST implement the specified error recovery behavior when it encounters t
 | Invalid `starting_token` | 10 ABORTED | Indicates that `starting_token` is not valid. | Caller SHOULD start the `ListSnapshots` operation again with an empty `starting_token`. |
 
 
+#### `ControllerExpandVolume`
+
+A Controller plugin MUST implement this RPC call if plugin has `EXPAND_VOLUME` controller capability.
+This RPC allows the CO to expand the size of a volume.
+
+This call can be made by the CO during any time in the lifecycle of the volume after creation if plugin has `EXPANDABLE_ONLINE` capability.
+If plugin has `EXPAND_VOLUME` node capability, then `NodeExpandVolume` MUST be called after `ControllerExpandVolume` has completed.
+
+
+`ControllerExpandVolume` MUST be called after `ControllerUnpublishVolume` if plugin has only `EXPANDABLE_OFFLINE` capability.
```

Comment:
This is not precise. I think it can also be called if the volume is provisioned, but not yet be published to any node.  I would say:

If the plugin has only `OFFLINE` volume expansion capability, `ControllerExpandVolume` MUST be called after `ControllerUnpublishVolume` is called and is successful if the volume is currently published to a node.